PresentScreenings/Views/ColorView.cs

```diff
@@ -176,6 +168,67 @@ public static void DrawTicketAvalabilityFrame(CGContext g, Screening screening,
         {
             DrawTicketAvalabilityFrame(g, screening, new CGRect(0, 0, side, side));
         }
+
+        public static void DrawOnDemandAvailabilityStatus(CGContext g, OnDemandScreening screening, CGRect rect, bool selected)
+        {
+            // Establish some base dimensions.
+            const int maxDaysInRuler = 5;
+            const int margin = 2;
+            nfloat w = rect.Width - 2 * margin;
+            nfloat h = rect.Height - 2 * margin;
+            nfloat x = rect.X + margin;
+            nfloat y = rect.Y + margin;
```

Comment:
Please consider to replace one-letter names into descriptive ones.

Reply:
Variable names `x` and `y` are generally accepted.
I propose to accept `w` for width and `h` for height throughout the project, also in combinations like `wBar`.
Did replace `g` into `context` though.

Reply:
OK, but please put that in the Agreements.

Reply:
Done.